bin/sendsketch_to_prokka.py

```diff
@@ -9,22 +9,30 @@
 genus = "Multiple"
 
 with open(sketch_file, "r") as sketch:
-	sketch_lines = sketch.readlines()
-
-	genus_one_line = sketch_lines[3]
-	genus_one = genus_one_line.split("\t")[11].split(" ")[0]
-	species_one = genus_one_line.split("\t")[11].split(" ")[1]
-
-	genus_two_line = sketch_lines[4]
-	genus_two = genus_two_line.split("\t")[11].split(" ")[0]
-
-	if len(genus_two) == 0 or genus_one == genus_two:
-		genus = genus_one
-		output_species = species_one.rstrip()
-		with open(stain_file, "r") as stain:
-			for line in stain:
-				if line.split("\t")[0] == genus:
-					output_stain = line.rstrip().split("\t")[1]
-	else:
-		output_stain = "Contaminated"
-print(output_stain+"\t"+genus+"\t"+output_species)
+    sketch_lines = sketch.readlines()
+
+    genus_one_line = sketch_lines[3]
+    genus_one = genus_one_line.split("\t")[11].split(" ")[0]
+    species_one = genus_one_line.split("\t")[11].split(" ")[1]
+
+    genus_two_line = sketch_lines[4]
+    genus_two = genus_two_line.split("\t")[11].split(" ")[0]
+
+    if len(genus_two) == 0 or genus_one == genus_two:
+        genus = genus_one
+        output_species = species_one.rstrip()
+        with open(stain_file, "r") as stain:
+            for line in stain:
+                if line.split("\t")[0] == genus:
+                    output_stain = line.rstrip().split("\t")[1]
+    else:
+        output_stain = "Contaminated"
+
+
+with open("sendsketch_to_prokka_output_stain.txt", "w") as f:
+    f.writelines(output_stain)
```

Comment:
I am not sure why, but the indentation seemed to be a bit off in this branch.

It caused the `SCREEN...` process to fail.

Could you please confirm both these scripts once?